systemuser.sh

```diff
@@ -87,6 +87,12 @@ echo "c.ContentsManager.checkpoints_class = 'swancontents.filemanager.checkpoint
 # Fixes issue with frozen servers with async io errors, fix from https://github.com/jupyter/notebook/issues/6164
 echo "c.NotebookApp.kernel_manager_class = 'notebook.services.kernels.kernelmanager.AsyncMappingKernelManager'" >> $JPY_CONFIG
 
+# Disable pinging for maintenance notifications when running on Kubernetes
```

Comment:
Just cosmetics but: can we not place this in the middle of the generation of the notebook config file?

Reply:
done